scripts/rust/tests.lint.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+set -xue
+
+if ! [[ "$0" =~ scripts/rust/tests.lint.sh ]]; then
+  echo "must be run from repository root"
+  exit 255
+fi
+
+# https://rust-lang.github.io/rustup/installation/index.html
+# rustup toolchain install nightly --allow-downgrade --profile minimal --component clippy
+#
+# https://github.com/rust-lang/rustfmt
+# rustup component add rustfmt
+# rustup component add rustfmt --toolchain nightly
+# rustup component add clippy
+# rustup component add clippy --toolchain nightly
+
+rustup default stable
+cargo fmt --all --verbose -- --check
+
+rustup default nightly
+cargo +nightly fmt --all -- --config-path .rustfmt.nightly.toml --verbose --check || true
```

Comment:
@samliok if we drop the `|| true` here these will run/error can you see if the output is reasonable if yes update the script? If not add a TODO to enable?